### DISCUSSION TITLE
User Guide documentation

### DISCUSSION THREAD

**Discussion by floradanna** [seek4science/seek#1127] [General]
Hi all, I was wondering if the Community Workers could improve the current FAIRDOM-SEEK User Guide. I noticed that few features are missing and that some items are listed without a clear order.

I would like to propose some changes, such as grouping items by the categories as shown in "Create/Browse", that are Yellow Pages, Experiments, Assets, Samples etc...

Here there is a first draft, please let me know what's your opinion.

**USER GUIDE**

YOUR PROFILE
* [Registering in SEEK](https://docs.seek4science.org/help/user-guide/registering.html)
* [Logging into SEEK](https://docs.seek4science.org/help/user-guide/login.html)
    * [Logging into SEEK via ELIXIR AAI](https://docs.seek4science.org/help/user-guide/aai.html)
* [Managing your account](https://docs.seek4science.org/help/user-guide/managing-account.html)
    * [Managing your identities/alternative login methods](https://docs.seek4science.org/help/user-guide/managing-identities.html)
* [Editing your profile](https://docs.seek4science.org/help/user-guide/editing-profile.html)
* Favourites (missing)

YELLOW PAGES
* [Browsing content](https://docs.seek4science.org/help/user-guide/browsing.html)
* [Creating and managing your own Programmes](https://docs.seek4science.org/help/user-guide/programme-creation-and-management.html)
* [Creating a Project](https://docs.seek4science.org/help/user-guide/create-a-project.html)
* [Joining a Project](https://docs.seek4science.org/help/user-guide/join-a-project.html)
* [Project Dashboard](https://docs.seek4science.org/help/user-guide/project-dashboard.html)
* Viewing Project in Single Page ([draft ongoing](https://github.com/ELIXIR-Belgium/DataHub-help/pull/5))
* [Specialist user roles](https://docs.seek4science.org/help/user-guide/roles.html) and their capabilities
    * [Programme administrator](https://docs.seek4science.org/help/user-guide/roles.html#programme-administrator)
    * [Project administrator](https://docs.seek4science.org/help/user-guide/roles.html#project-administrator)
        * [Assign people to Project roles](https://docs.seek4science.org/help/user-guide/administer-project-members.html#assign-people-to-project-roles)
        * [Flag when a person becomes inactive in a Project](https://docs.seek4science.org/help/user-guide/administer-project-members.html#flag-when-a-person-becomes-inactive-in-a-project) (still coming soon?)
    * [Create Organisms](https://docs.seek4science.org/help/user-guide/adding-admin-items.html#creating-organisms)
    * [Create Profiles](https://docs.seek4science.org/help/user-guide/adding-admin-items.html#creating-profiles)
    * [Create Institutions](https://docs.seek4science.org/help/user-guide/adding-admin-items.html#creating-institutions)
    * [Add and remove people from a Project](https://docs.seek4science.org/help/user-guide/administer-project-members.html#add-and-remove-people-from-a-project)
    * [Asset housekeeper](https://docs.seek4science.org/help/user-guide/roles.html#asset-housekeeper)
    * [Asset gatekeeper](https://docs.seek4science.org/help/user-guide/roles.html#asset-gatekeeper)

EXPERIMENTS
* [Generating the ISA structure](https://docs.seek4science.org/help/user-guide/generating-the-isa-structure.html)
    * [Creating an Investigation](https://docs.seek4science.org/help/user-guide/generating-the-isa-structure.html#creating-an-investigation)
    * [Creating a Study](https://docs.seek4science.org/help/user-guide/generating-the-isa-structure.html#creating-a-study)
        * [Investigation details](https://docs.seek4science.org/help/user-guide/general-attributes.html#investigation-details)
    * [Creating an Experimental assay](https://docs.seek4science.org/help/user-guide/generating-the-isa-structure.html#creating-an-experimental-assay)
        * [Assay Type](https://docs.seek4science.org/help/user-guide/general-attributes.html#assay-type)
        * [Technology Type](https://docs.seek4science.org/help/user-guide/general-attributes.html#technology-type)
    * [Creating a Modelling analysis](https://docs.seek4science.org/help/user-guide/generating-the-isa-structure.html#creating-a-modelling-analysis)
        * [Biological Problem Addressed](https://docs.seek4science.org/help/user-guide/general-attributes.html#biological-problem-addressed)
* [ISA Overview Graph](https://docs.seek4science.org/help/user-guide/isa-overview.html)
* [Experimental assays and Modelling analysis](https://docs.seek4science.org/help/user-guide/general-attributes.html#experimental-assays-and-modelling-analysis)
    * [Study](https://docs.seek4science.org/help/user-guide/general-attributes.html#study)
* Designing experiments in Single Page ([draft ongoing](https://github.com/ELIXIR-Belgium/DataHub-help/pull/2))
    * Creating an Investigation in Single Page
    * Creating a Study in Single Page
    * Creating an Assay in Single Page
    * Creating samples in Single Page
* Export ISA (to do)
* [Making an Investigation, Study or Assay citable](https://docs.seek4science.org/help/user-guide/investigation-snapshots.html)
    * [Making public](https://docs.seek4science.org/help/user-guide/investigation-snapshots.html#making-public)
    * [Snapshotting](https://docs.seek4science.org/help/user-guide/investigation-snapshots.html#snapshotting)
    * [Creating a Research Object](https://docs.seek4science.org/help/user-guide/investigation-snapshots.html#creating-a-research-object)
    * [Assigning a DOI](https://docs.seek4science.org/help/user-guide/investigation-snapshots.html#assigning-a-doi)

ASSETS
* [Adding assets (data, models, SOPs, publications) to SEEK](https://docs.seek4science.org/help/user-guide/adding-assets.html)
* [Data Files](https://docs.seek4science.org/help/user-guide/general-attributes.html#data-files)
    * Data file wizard(https://docs.seek4science.org/help/user-guide/data-file-upload-wizard.html)
* Model (missing page) --> available in API documentation (https://fairdomhub.org/api#tag/models)
    * [Simulate a model on JWS Online](https://docs.seek4science.org/help/user-guide/simulate-on-jws-online.html)
* [SOPs](https://docs.seek4science.org/help/user-guide/general-attributes.html#sops)
* [Publications](https://docs.seek4science.org/help/user-guide/general-attributes.html#publications)
* [Collections](https://docs.seek4science.org/help/user-guide/collections.html)
* Document
* Workflow
* File template
* Placeholder
* [Creating new asset versions](https://docs.seek4science.org/help/user-guide/uploading-new-versions.html)
    * [Comparing two versions of a Model](https://docs.seek4science.org/help/user-guide/model-comparison.html)

SAMPLES
* [Samples](https://docs.seek4science.org/help/user-guide/samples.html)
    * [Create a Sample Type](https://docs.seek4science.org/help/user-guide/create-sample-type.html)
    * [Create a Sample](https://docs.seek4science.org/help/user-guide/create-sample.html)
    * [Legacy Biosamples](https://docs.seek4science.org/help/user-guide/legacy-biosamples.html)
* Creating samples in Single Page (link to subpage above)
* Templates (to do)
    * Sample Type in Single Page
* Query Samples (to do)
    * By Sample Type
    * By Templates

ACTIVITES
Events (link below?)
Presentation (missing)

GENERAL ATTRIBUTES AND LINKS
* [General attributes and links](https://docs.seek4science.org/help/user-guide/general-attributes.html) (name it “Generic linking variables in SEEK”)
    * [Title](https://docs.seek4science.org/help/user-guide/general-attributes.html#title)
    * [Description](https://docs.seek4science.org/help/user-guide/general-attributes.html#description)
    * [Projects](https://docs.seek4science.org/help/user-guide/general-attributes.html#projects)
    * [Sharing](https://docs.seek4science.org/help/user-guide/general-attributes.html#sharing)
        * [Bulk changing of sharing permissions](https://docs.seek4science.org/help/user-guide/bulk-change-sharing-permission.html)
    * [Tags](https://docs.seek4science.org/help/user-guide/general-attributes.html#tags)
    * [Organisms](https://docs.seek4science.org/help/user-guide/general-attributes.html#organisms)
    * STRAIN???
    * [Attributions](https://docs.seek4science.org/help/user-guide/general-attributes.html#attributions)
    * [Creators](https://docs.seek4science.org/help/user-guide/general-attributes.html#creators)
    * [Experimentalists](https://docs.seek4science.org/help/user-guide/general-attributes.html#experimentalists)
    * [Events](https://docs.seek4science.org/help/user-guide/general-attributes.html#events)
    * PRESENTATION?
    * Discussion channels (missing)
* [Licenses](https://docs.seek4science.org/help/user-guide/licenses.html)

INTEGRATIONS
* [Using SEEK with openBIS](https://docs.seek4science.org/help/user-guide/openbis.html)
* [Using Copasi in SEEK](https://docs.seek4science.org/help/user-guide/copasi-button.html)

USER GUIDE FOR API
* [API Introduction](https://docs.seek4science.org/help/user-guide/api.html)

CONTRIBUTE
* [Reporting a bug or feature request](https://docs.seek4science.org/tech/reporting-bugs-and-features.html)
* [Contributing to these SEEK Documents](https://docs.seek4science.org/contributing.html)

**Comment by floradanna** [comment]
Missing information could be taken from API documentation: https://fairdomhub.org/api#tag/models

**Reply by OlgaKrebs** [reply]
thanks, Flora for your suggestions (and pre-work)  I agree with the structure, just one objection : separation into EXPERIMENT, ASSET and SAMPLE (I know it is same as in  SEEK menu ) cause it is semantically confusing (especially EXPERIMENT part ). We can discuss this may be having one "superfolder" above these tree and looking for better naming

**Reply by OlgaKrebs** [reply]
... also "publishing with SEEK" can be an extra docu folder and not just part in ISA(EXPERIMENT) forlder - user very often ask where to find this

**Reply by floradanna** [reply]
Indeed, the "publishing" is missing or not very visible. By publishing, you mean
- make an item public (I have a doubt: is setting sharing permission as Visible to Public the same of clicking "publish" to create DOI?)
- make snapshot
- assign DOI
Correct? This is indeed described only for ISA, but it is possible for any items.

We could add it to:
GENERAL ATTRIBUTES AND LINKS

* [General attributes and links](https://docs.seek4science.org/help/user-guide/general-attributes.html) (name it “Generic linking variables in SEEK”)
  * [Title](https://docs.seek4science.org/help/user-guide/general-attributes.html#title)
  * [Description](https://docs.seek4science.org/help/user-guide/general-attributes.html#description)
  * [Projects](https://docs.seek4science.org/help/user-guide/general-attributes.html#projects)
  * [Tags](https://docs.seek4science.org/help/user-guide/general-attributes.html#tags)
  * [Organisms](https://docs.seek4science.org/help/user-guide/general-attributes.html#organisms)
  * STRAIN???
  * [Attributions](https://docs.seek4science.org/help/user-guide/general-attributes.html#attributions)
  * [Creators](https://docs.seek4science.org/help/user-guide/general-attributes.html#creators)
  * [Experimentalists](https://docs.seek4science.org/help/user-guide/general-attributes.html#experimentalists)
  * [Events](https://docs.seek4science.org/help/user-guide/general-attributes.html#events)
  * PRESENTATION?
  * Discussion channels (missing)
* [Licenses](https://docs.seek4science.org/help/user-guide/licenses.html)
* [Sharing](https://docs.seek4science.org/help/user-guide/general-attributes.html#sharing)
  * [Bulk changing of sharing permissions](https://docs.seek4science.org/help/user-guide/bulk-change-sharing-permission.html)
* PUBLISHING ITEMS
  * [Making an Investigation, Study or Assay citable](https://docs.seek4science.org/help/user-guide/investigation-snapshots.html)
  * MAKING ITEMS CITABLE

**Reply by OlgaKrebs** [reply]
what is about having sharing and publishing in one "big" folder with a title "Sharing Permissions for your data : from privat to public" and inside 1. sharing 2. bulk sharing 3. publishing with and without DOI 4. 4. RO & Zenodo 5.licensing

**Reply by floradanna** [reply]
Yes, it could be a good structure. 

I don't find any documentation about RO and Zenodo. Is it somewhere I don't know about? If it is missing, could you write the "step by step" process to generate RO and Depositing in  Zenodo? You can either do it GitHub pages or in a google doc and share the link here.

**Reply by abecam** [reply]
So related to that, the document for the short description and links to documentation within Seek is here: [Short description for Pop-Up information](https://docs.google.com/document/d/1FYwkpRwrgcfICUjnEOph78Hn6ITsyZbJkQ8UCaUC_50/edit?usp=sharing)
I sometime provided the description from the documentation  (together with the link to it then) and/or from the API doc (so without link, as there are no further information there).
Some items are still without anything.

**Comment by floradanna** [comment]
@abecam Should we add the missing content to the doc?

**Reply by abecam** [reply]
Yes, and don't hesitate to edit (also as there are 2 choices when there was entry in both the documentation and the API doc).

**Comment by floradanna** [comment]
We had to provide documentation for DataHub urgently for our test instance, I re-arranged and added content to the User Guide here: https://datahub.help.elixir-belgium.org/test/help/user-guide/index.html 

We could discuss what changes can be merged and made available in the main code and what needs to be edited further.

**Comment by floradanna** [comment]
Structure in [User Guide pages](https://datahub.help.elixir-belgium.org/test/help/user-guide/index.html)

**Comment by rabuono** [comment]
Highlighting a comment:
Having the guide pages without basic context and definition of concepts might confuse some users.

I think that keeping the user guides separate from the basic concepts, as it is in[ FAIRDOMhub now](https://docs.seek4science.org/help/), is a good idea. One can present the basic concepts above the user guide, but is not committed to having concepts that don't apply to the instance in the help section

**Comment by stuzart** [comment]
I've created a branch for you here: https://github.com/seek4science/seek/tree/gh-pages-refactor

`gh-pages-refactor`

**Reply by stuzart** [reply]
> I've created a branch for you here: https://github.com/seek4science/seek/tree/gh-pages-refactor
> 
> `gh-pages-refactor`

It turns out I got my wires crossed and this branch isn't needed, so will be deleted.
